src-docs/src/views/elastic_charts/accessibility_example.js

```diff
@@ -0,0 +1,313 @@
+import React from 'react';
+import {
+  EuiCallOut,
+  EuiCode,
+  EuiCodeBlock,
+  EuiSpacer,
+  EuiText,
+} from '../../../../src/components';
+import { GuideSectionTypes } from '../../components';
+import { ExternalBadge } from './shared';
+import { TextureMultiSeriesChart } from './texture';
+import { TexturedStylesProps, ChartProps } from './texture_props';
+
+const TextureMultiSeriesChartSource = require('!!raw-loader!./texture');
+
+export const ElasticChartsAccessibilityExample = {
+  title: 'Accessibility features',
+  intro: (
+    <>
+      <ExternalBadge />
+      <EuiSpacer size="l" />
+      <EuiText>
+        <p>
+          Elastic charts is becoming more and more accessible for all users.
+        </p>
+        <p>
+          <strong>Available a11y-related features</strong>
+        </p>
+        <ul>
+          <li>Chart titles</li>
+          <li>Chart descriptions</li>
+          <li>
+            Chart data in tabular form<sup>*</sup>
+          </li>
+          <li>
+            Texture fill<sup>*</sup>
+          </li>
+          <li>
+            Semantic groupings<sup>*</sup>
+          </li>
+        </ul>
+        <p>
+          <small>
+            <sup>*</sup>Not all chart types that may benefit from this feature
+            have it implemented as of yet
+          </small>
+        </p>
+      </EuiText>
+    </>
+  ),
+  sections: [
+    {
+      title: 'Titles',
```

Comment:
I changed all the titles to be what they do instead of the prop name, thinking that someone might know "I want a title" but not "I want to set ariaLabelledBy"